### ISSUE TITLE
Ruffle Desktop App Bugs

### ISSUE THREAD

**Issue by uynnoangabriel** [ruffle-rs/ruffle#18371]
### Describe the bug

When i play the .swf file on Ruffle Desktop App, my app go to a all black screen, and i cant see whats playing in the background (but if the .swf have a background sound, i listen but i even cant see whats happening). Please fix it the almost fast possible.

### Expected behavior

~Play the content

### Content Location

PC Files

### Affected platform

Desktop app

### Operating system

Windows 10

### Browser

Microsoft Edge

**Comment by Croworbit** [comment]
you'll need to send the swf file you're having trouble with, or it will be difficult to fix your problem

after you select your swf, are you pressing start/open?

**Comment by uynnoangabriel** [comment]
yes sir, im pressing start/open, but the emulator starts a black screen, i
think the swf icons is merged.
I you upload a link here with one of the games im trying to play, but im
having this problem;
https://github.com/uynnoangabriel/noahxyz/raw/ab7c4d063db802fc190fbb84e541dd299db79fd3/coraline-and-the-secret-door.swf

Em qui., 24 de out. de 2024 10:13 AM, Croworbit ***@***.***>
escreveu:
